packages/headless/src/features/debug/ranking-info-parser.ts

```diff
@@ -266,7 +266,7 @@ const parseTermsWeights = (
 
 const parseQREWeights = (value: string): QueryRankingExpressionWeights[] => {
   const REGEX_EXTRACT_QRE_WEIGHTS =
-    /(Expression:\s".*")\sScore:\s(?!0)([0-9]+)\n+/g;
+    /(Expression:\s".*")\sScore:\s(?!0)([-0-9]+)\n+/g;
```

Comment:
This would allow for a negative symbol in between or at the end of numbers.

```suggestion
    /(Expression:\s".*")\sScore:\s-?(?!0)([0-9]+)\n+/g;
```

Reply:
commited and the tests failed 😢